GitHubProjectColumns.ps1

```diff
@@ -0,0 +1,381 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+function Get-GitHubProjectColumn
+{
+<#
+    .DESCRIPTION
+        Get the columns for a given Github Project.
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER Project
+        Id of the project to retrieve a list of columns for.
```

Comment:
nit: As a variable, it would be `Id`.  When mentioning it though, it's `ID` because it's an abbreviation.  Here and elsewhere.